src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -973,6 +973,262 @@ function Test-ApplicationGatewayCRUDRewriteRuleSetWithConditions
 	}
 }
 
+function Test-ApplicationGatewayCRUDRewriteRuleSetWithUrlConfiguration
```

Comment:
Playback can not be done since the manifest for 2019-01-11 is not updated

Reply:
When will it be available?

Reply:
Added

Reply:
added